sdk/storage/azure-storage-file-datalake/tests/test_file.py

```diff
@@ -34,7 +34,7 @@ class FileTest(StorageTestCase):
     def setUp(self):
         super(FileTest, self).setUp()
         url = self._get_account_url()
-        self.dsc = DataLakeServiceClient(url, credential=self.settings.STORAGE_DATA_LAKE_ACCOUNT_KEY)
+        self.dsc = DataLakeServiceClient(url, credential=self.settings.STORAGE_DATA_LAKE_ACCOUNT_KEY, logging_enable=True)
```

Comment:
is logging_enable set to True on purpose?

Reply:
yeah, I turned that on on purpose. so that if in real_settings.py we set `ENABLE_LOGGING=True` we can see the request/response info